src/components/02-organisms/Attachments/FileDownloader.js

```diff
@@ -0,0 +1,53 @@
+import React, { useContext, useMemo } from "react";
+import { injectIntl } from "react-intl";
+import PropTypes from "prop-types";
+import FetchHandler from "../../../FetchHandler";
+import { BackendContext } from "../../../context/backendContext";
+// import { BackendContext } from "./context/backendContext";
+
+const FileDownloader = props => {
+  const { state: fromBackend = {}, dispatch } = useContext(BackendContext);
+
+  const fetchSetup = useMemo(() => {
+    console.log(props.uuid);
+    if (props.uuid)
+      return [
+        {
+          key: "liitteet",
+          dispatchFn: dispatch,
+          urlEnding: props.uuid
+        }
+      ];
+    else return [];
+  }, [dispatch, props.file]);
```

Comment:
props.uuid is missing from dependency array. props.file should be removed.

Reply:
Argh, changed the prop used.